.changes/next-release/bugfix-AWSSDKforJavav2-eeb4182.json

```diff
@@ -0,0 +1,6 @@
Bucket\nPutBucketInventoryConfigurationRequest.Id\nPutBucketInventoryConfigurationRequest.InventoryConfiguration\nPutBucketLifecycleConfigurationRequest.Bucket\nPutBucketLoggingRequest.BucketLoggingStatus\nPutBucketLoggingRequest.Bucket\nPutBucketMetricsConfigurationRequest.Bucket\nPutBucketMetricsConfigurationRequest.Id\nPutBucketMetricsConfigurationRequest.MetricsConfiguration\nPutBucketNotificationConfigurationRequest.Bucket\nPutBucketNotificationConfigurationRequest.NotificationConfiguration\nPutBucketOwnershipControlsRequest.Bucket\nPutBucketOwnershipControlsRequest.OwnershipControls\nPutBucketPolicyRequest.Bucket\nPutBucketPolicyRequest.Policy\nPutBucketReplicationRequest.Bucket\nPutBucketReplicationRequest.ReplicationConfiguration\nPutBucketRequestPaymentRequest.Bucket\nPutBucketRequestPaymentRequest.RequestPaymentConfiguration\nPutBucketTaggingRequest.Bucket\nPutBucketTaggingRequest.Tagging\nPutBucketVersioningRequest.Bucket\nPutBucketVersioningRequest.VersioningConfiguration\nPutBucketWebsiteRequest.Bucket\nPutBucketWebsiteRequest.WebsiteConfiguration\nPutObjectAclRequest.Bucket\nPutObjectAclRequest.Key\nPutObjectLegalHoldRequest.Bucket\nPutObjectLegalHoldRequest.Key\nPutObjectLockConfigurationRequest.Bucket\nPutObjectRequest.Bucket\nPutObjectRequest.Key\nPutObjectRetentionRequest.Bucket\nPutObjectRetentionRequest.Key\nPutObjectTaggingRequest.Bucket\nPutObjectTaggingRequest.Key\nPutObjectTaggingRequest.Tagging\nPutPublicAccessBlockRequest.Bucket\nPutPublicAccessBlockRequest.PublicAccessBlockConfiguration\nQueueConfiguration.Events\nQueueConfiguration.QueueArn\nRedirectAllRequestsTo.HostName\nReplicaModifications.Status\nReplicationConfiguration.Role\nReplicationConfiguration.Rules\nReplicationRule.Destination\nReplicationRule.Status\nReplicationTime.Status\nReplicationTime.Time\nRequestPaymentConfiguration.Payer\nRestoreObjectRequest.Bucket\nRestoreObjectRequest.Key\nRoutingRule.Redirect\nS3Location.BucketName\nS3Location.Prefix\nSSEKMS.KeyId\nSelectObjectContentRequest.Bucket\nSelectObjectContentRequest.ExpressionType\nSelectObjectContentRequest.Expression\nSelectObjectContentRequest.InputSerialization\nSelectObjectContentRequest.Key\nSelectObjectContentRequest.OutputSerialization\nSelectParameters.ExpressionType\nSelectParameters.Expression\nSelectParameters.InputSerialization\nSelectParameters.OutputSerialization\nServerSideEncryptionByDefault.SSEAlgorithm\nServerSideEncryptionConfiguration.Rules\nSseKmsEncryptedObjects.Status\nStorageClassAnalysisDataExport.Destination\nStorageClassAnalysisDataExport.OutputSchemaVersion\nTag.Key\nTag.Value\nTagging.TagSet\nTiering.AccessTier\nTiering.Days\nTopicConfiguration.Events\nTopicConfiguration.TopicArn\nUploadPartCopyRequest.CopySource\nUploadPartCopyRequest.PartNumber\nUploadPartCopyRequest.UploadId\nUploadPartRequest.Bucket\nUploadPartRequest.Key\nUploadPartRequest.PartNumber\nUploadPartRequest.UploadId\nWriteGetObjectResponseRequest.RequestRoute\nWriteGetObjectResponseRequest.RequestToken"
```

Comment:
Woo, that's long. I didn't realize how many operations. That might fill up our changelog and make it hard to read. Should we just say that we'd verify it's set for required query parameters and leave it at that?